examples/plotting/magnetogram_active_regions.py

```diff
@@ -8,53 +8,31 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-import astropy.units as u
 from astropy.coordinates import SkyCoord
-from astropy.time import TimeDelta
 
 import sunpy.coordinates
 import sunpy.data.sample
 import sunpy.map
 from sunpy.io.special import srs
-from sunpy.net import Fido
-from sunpy.net import attrs as a
-from sunpy.time import parse_time
 
 ##############################################################################
-# For this example, we will start with the sample data. We need an HMI
-# file and use it to create a map.
+# For this example, we will start with the sample data. We need an HMI file and
+# use it to create a map, and the SRS table which contains a list of active
+# regions. Both of these data can be downloaded with ``Fido``.
 
 smap = sunpy.map.Map(sunpy.data.sample.HMI_LOS_IMAGE)
-
-##############################################################################
-# Then using the observation time of the sample image we will download the corresponding
-# NOAA SWPC solar region summary. These are published once a day, so we will define an
-# ``end_time`` that ends just before the next day, otherwise you will download two files.
-
-start_time = parse_time(smap.date)
-end_time = start_time + TimeDelta(23*u.hour + 59*u.minute + 59*u.second)
-
-##############################################################################
-#  Here we use `sunpy.net.Fido` to acquire the data.
-
-srs_results = Fido.search(a.Time(start_time, end_time), a.Instrument.srs_table)
-srs_downloaded_files = Fido.fetch(srs_results)
-
-##############################################################################
-# To read this file, we pass the filename into the SRS reader. So now
-# ``srs_table`` contains an astropy table.
-srs_table = srs.read_srs(srs_downloaded_files[0])
-print(srs_table)
+srs_table = srs.read_srs(sunpy.data.sample.SRS_TABLE)
 
 ##############################################################################
 # We only need the rows which have 'ID' = 'I' or 'IA'.
+# Some tables do not have these columns, so we exit the script if they are not
+# present.
 
 if 'I' in srs_table['ID'] or 'IA' in srs_table['ID']:
     srs_table = srs_table[np.logical_or(srs_table['ID'] == 'I',
                                         srs_table['ID'] == 'IA')]
 else:
-    print("Warning : No I or IA entries for this date.")
-    srs_table = None
+    raise ValueError("No I or IA entries for this date.")
```

Comment:
Do we will need this?

Reply:
:man_shrugging: It feels better to spell out that some of the files don't have the columns if we are expecting people to edit this script to their own dates.

Reply:
Sure.